src/mpl2/src/hier_rtlmp.cpp

```diff
@@ -260,9 +260,13 @@ void HierRTLMP::setDefaultThresholds()
     } else {
       congestion_weight_ = 1.0 * snap_layer_ / layers.size();
     }
-    logger_->report("snap_layer : {}  congestion_weight : {}",
-                    snap_layer_,
-                    congestion_weight_);
+    debugPrint(logger_,
+               MPL,
+               "macro_plamcement",
```

Comment:
Thoughout: macro_plamcement -> macro_placement

Reply:
I followed some other debug prints that were already there. Should I create another group? If so, how can I do that? I looked but didn't found where I can create it.

Reply:
It just looks like a typo.  I don't think it is meant to be a separate group.

Reply:
Ah now I see the typo